lib/Doctrine/Connection/Sqlite.php

```diff
@@ -38,6 +38,11 @@ class Doctrine_Connection_Sqlite extends Doctrine_Connection_Common
      */
     protected $driverName = 'Sqlite';
 
+    /**
+     * @var PDO  $dbh                                the database handler
+     */
+    protected $dbh;
```

Comment:
Defining the connection object here to make it more specific to this connection type ($dbh in Doctrine_Connection is `protected` but docblock has it as `PDO|Doctrine_Adapter_Interface`).